assets/README.md

```diff
@@ -0,0 +1,36 @@
+# Mayflower-Tokens
+[![npm package][npm-badge]][npm]
+
+Mayflower-tokens provides a bundled up library of design tokens and assets that you will need for theming your front end framework to look and feel like [Mass.gov](https://mass.gov) and other Commonwealth web properties that are in Mayflower ecosystem. Mayflower-tokens includes color token SCSS variables, fonts, icons and other imagery from Mayflower — the [enterprise design system for the Commonwealth of Massachusetts][mayflower-doc].
+
+- **Codebase:** [Mayflower monorepo][mayflower-github] `assets` subdirectory
+  >[Mayflower monorepo][mayflower-github] comprised of [Mayflower documentation][mayflower-doc], two component libraries — [Mayflower React][react-storybook] and [Mayflower PatternLab][patternLab], and their [shared assets][shared-assets].
+  > Refer to [Mayflower PatternLab Static Site][patternlab] for the set of UI components consumed in Mass.gov.
+- **Live demo:** [mayflower-react storybook][react-storybook]
+
+## Using Mayflower-Tokens in Your Project
+1. Install mayflower-tokens into your project as a dependency.
+`npm i @massds/mayflower-tokens --save`
+2. Import color tokens into your SASS/SCSS file
+`@import '[path to node_modules]/@massds/mayflower-tokens/colors/mayflower-color_tokens';`
```

Comment:
Could there be a root file that exports all the tokens, so this import path is a bit clearer? For example:

```
@import '[path to node_modules]/@massds/mayflower-tokens/index'
```

Reply:
It also feels like the path to individual variable files could be simpler, for example:

```
@import '[path to node_modules]/@massds/mayflower-tokens/colors'
```

Reply:
I think if people are using node-sass, they can just import the package like `@import '@massds/mayflower-tokens'. I set the entry point for style in package.json
```
  "style": "colors/_mayflower-color-tokens.scss",
```
But I agree it can be simplified, should I move it out of the folder `colors/` and rename it `colors.scss`? I wonder if we should indicate that its a partial only contains variables, maybe it's not necessary as it's the only scss file